search-sentiments-map/public/scripts/current-data-script.js

```diff
@@ -34,7 +53,17 @@ function getCurrentSearchData() {
  * @param {string} trend New trend to get data for.
  */
 function setNewTrend(trend) {
-  fetch('/search/' + trend)
+  currentTrend = trend;
+
+  // Bold and italicize the currently viewed trend.
+  let trendElements = document.getElementById('trends-list').childNodes;
+  trendElements.forEach(function(trendElement) {
+    trendElement.innerHTML = (trendElement.innerText === currentTrend) ?
+        '<span class="font-weight-bold font-italic">' + currentTrend +
+        '</span>' : trendElement.innerHTML = trendElement.innerText;
+  })
```

Comment:
Are you sure this works as intended? First, do you set it back to be clickable anywhere after another trend is clicked? Also, I think we should set it in the onclick method of each trendElement so that we don't need to loop over all the elements again

Reply:
maybe just add a tag for bold or a class, instead of changing the entire innerHTML

Reply:
So I don't change whether or not is it clickable. It remains clickable throughout. Another thing is that new trends can be changed not just by clicking. For example changing the timeline also changes the trend, so I think it is best to handle the changes here. We are only adding one span with bold not changing all of the HTML.

Reply:
Ok I see what you are doing here. Makes sense